### PR TITLE
1643 public related terms

### PR DESCRIPTION
**Pull request by ledsoft**
https://kbss.felk.cvut.cz/redmine/issues/1643

Requires https://github.com/kbss-cvut/termit/pull/121 to be merged and deployed for full functionality.

### CODE REVIEW

File: src/action/AsyncActions.ts [hunk 1]
```diff
@@ -153,16 +179,16 @@ export function loadVocabulary(
...
-) {
+function loadImportedVocabulariesIntoState(vocabularyIri: IRI) {
   const action = {
     type: ActionType.LOAD_VOCABULARY_IMPORTS,
   };
   return (dispatch: ThunkDispatch, getState: GetStoreState) => {
     dispatch(asyncActionRequest(action, true));
     return Ajax.get(
-      `${getApiPrefix(getState())}/vocabularies/${vocabularyIri.fragment}/imports`,
+      `${getApiPrefix(getState())}/vocabularies/${
```
Comment (psiotwo) [inline src/action/AsyncActions.ts:189]: Not a big deal, but the reformatting looks a bit strange - here it is split to multiple rows, some lines up and down the methods params are put to a single row ...

**Review by psiotwo — COMMENTED** [review]
Codewise it looks like just adding the getApiPrefix method and the rest just reformatting?

File: src/action/AsyncActions.ts (hunk 1 above)
Comment (ledsoft) [reply src/action/AsyncActions.ts:189]: If you do not like the formatting, blame Prettier. I had nothing to do with it whatsoever.

**Comment by ledsoft** [comment]
Backend deployed to dev, ready for testing.

**Review by psiotwo — APPROVED** [review]
lgtm

**Pull request merged**